### ISSUE TITLE
About training results

### ISSUE THREAD

**Issue by haoweiz23** [sgvaze/osr_closed_set_all_you_need#24]
Hi,

I found that the evaluation metrics are unstable among different epochs when I conducted experiments on the custom dataset. I would like to ask whether you report the last or the best epoch results in your training exps.

**Comment by sgvaze** [comment]
Hello, good question. 

I believe we took the model at the final epoch as not all datasets had a validation set. I would suggest tuning the hyper-parameters (e.g learning rate or LR schedule) to smooth the training.
